utils/preview.ts

```diff
@@ -0,0 +1,36 @@
+import * as express from 'express';
+
+// Types
+import { Request, Response } from 'express';
+
+// Modules
+import * as fs from 'fs';
+import * as path from 'path';
+import * as glob from 'glob';
+
+// App
+const app = express();
+const port: number = 3000;
+
+const QUICKSTART_PATH: string = process.argv[2];
+
+/*
+  Validates that the directory path provided exists
+*/
+fs.access(`../quickstarts/${QUICKSTART_PATH}`, (error: any) => {
```

Comment:
Is there a built in type for error here? I couldn't find the best way to handle it through search.